assets/scss/main.scss

```diff
@@ -16,3 +16,28 @@
   padding: 20px 20px 30px 20px;
   margin-bottom: 45px;
 }
+
+.nav-list {
+  column-count: 2;
+  column-gap: 2em;
+  list-style-type: decimal;
+  padding-left: 20px;
+  font-size: 16px;
+  line-height: 30px;
+}
+
+.govuk-related-items {
+  border-top: 10px solid $govuk-blue;
+  margin-top: $gutter;
+  padding-top: 5px;
+
+  .heading-medium {
+    margin-bottom: .5em;
+    margin-top: .3em;
+  }
+
+  li {
+    list-style-type: none;
+    margin-bottom: 10px;
+  }
+}
```

Comment:
Please ensure the landing pages look and behave well on mobile at present they don't appear to be.

Reply:
I've added mobile styling for the navigation list